sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/ResourceBase.java

```diff
@@ -102,4 +109,56 @@ protected static final boolean isAuthTokenInvalid(String authToken) {
   protected static final boolean isStringEmpty(String str) {
     return (str == null) || str.isEmpty();
   }
+
+  protected Response fetchRows(
+      StargateGrpc.StargateBlockingStub blockingStub,
+      int pageSizeParam,
+      String pageStateParam,
+      boolean raw,
+      String cql,
+      QueryOuterClass.Values.Builder values) {
+    QueryOuterClass.QueryParameters.Builder paramsB = parametersBuilderForLocalQuorum();
+    if (!isStringEmpty(pageStateParam)) {
+      // surely there must better way to make Protobuf accept plain old byte[]? But if not:
```

Comment:
I really think this is the best you can do here esp. because of the `ByteValue` wrapper.

Reply:
I just moved this code but got it, I'll remove the comment.